Code/Scripts/network_predictions.py

```diff
@@ -0,0 +1,129 @@
+import os
+from deep_lens_modeling import network_predictions
+from paltas import generate
+
+#define a function for a general case of n situations
+def Predictions(sample_num, path, config_path, image_path):
```

Comment:
Overall note: the two functions in this file could be condensed into one function (to avoid duplication). If you write one function that takes in the path to the data.tfrecord, and returns y_test, y_pred, ... for the lenses in that one data.tfrecord, you can just call that function one time for each test set to avoid code duplication.